backend/api/MQTT/MqttService.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Text.Json;
+﻿using System.Text;
+using System.Text.Json;
```

Comment:
Do we need to say that we use .Json as well when we say using System.Text?

Reply:
Yes, we do